### PR TITLE
Books checkout api

### PR DESCRIPTION
**Pull request by MichaelShingo**

### CODE REVIEW

**Comment by MichaelShingo** [comment]
I had to merge the books API branch and the checkout API branch into this single branch, because the database schema changes caused both of them to fail CI/CD checks.

File: pages/api/books/PATCH.ts
Comment (MichaelShingo) [inline pages/api/books/PATCH.ts:1]: The request should contain a list of IDs for region and genre, and a single ID for location.

File: pages/api/checkouts/GET.ts
Comment (MichaelShingo) [inline pages/api/checkouts/GET.ts:1]: Single findMany(), either finds all or filters by memberId and/or bookId, depending on the request body contents.

File: pages/api/checkouts/PATCH.ts
Comment (MichaelShingo) [inline pages/api/checkouts/PATCH.ts:1]: Checksin a book by updating checkinDate and available in a single transaction.

File: pages/api/checkouts/POST.ts
Comment (MichaelShingo) [inline pages/api/checkouts/POST.ts:1]: Checks out a book by creating a Checkout and updating Book.available in a single transaction.

File: pages/api/genre/POST.ts
Comment (MichaelShingo) [inline pages/api/genre/POST.ts:1]: Not sure if we need routes to create genre, location, region?

File: prisma/schema.prisma
Comment (MichaelShingo) [inline prisma/schema.prisma:1]: Updated the schema.

The location field in Book is now relational.

LibraryMember now has a isStaff Boolean field.

Added a Checkout model.

**Review by madaley1 — APPROVED** [review]
This all LG2M! I'm interested to see how our changes to the schema interact, but we're good to merge here!

**Pull request merged**